ego_odom_topic: "ego_odom"
ego_state_topic: "ego_state_estimation"
object_detection_topic: "object_detection"

world_frame: "odom"
control_command_topic: "/cmd_vel"
3d_lidar_topic: "/velodyne_points"
2d_lidar_topic: "/scan"
